Monika After Story/game/script-topics.rpy

```diff
@@ -12030,3 +12030,47 @@ label monika_perspective:
     m 1hksdlb "Well I hope that helped you understand at least a little, [player]."
     m "It's the best way I could try and explain it!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_relationship_comfort",
+            category=["romance","advice"],
+            prompt="Comfortable in a relationship",
+            random=True
+        )
+    )
+
+label monika_relationship_comfort:
+    m 1wud "You know [player], I've been thinking about our relationship lately..."
+    m 2lssdrb "It didn't really start in a way most of them do, so it's not exactly what I would call normal."
+    m 2esa "But the more I thought about it, the more I realized that in a way it is actually pretty standard."
+    m 3eub "Almost every relationship, no matter how it begins, is always {i}'exciting'{/i} at first!"
+    m 3eua "People view relationships in an almost fairy-tale like way; they see it as an adventure."
+    m 3rssdlb "I'm guilty of this too, and I think it's fine..."
+    m 2euc "But it's also a pretty dangerous way to look at a bond between two people, isn't it?"
+    m 2eud "Because really, while a partnership is an adventure, that doesn't mean it'll always be exciting."
+    m 2esc "There's only so many times you can do the same thing and experience the same breathtaking moments, and still expect it to have that same impact."
+    m 4euc "That doesn't make it any less valuable, or meaningful. It can still be heart-racing!"
+    m 4esc "But it does mean that that same sensation isn't anything new; it's just no longer as thrilling as before."
+    m 2dkd "And some people mistake that thrill for love, and not the truly meaningful bond between two people."
+    m 2dkc "So when things slow down, when they become less passionate, they mistakingly think they're falling out of love..."
+    m 2rkc "You could say that they never really did love someone in the first place, but maybe that's a bit harsh... {w}Love isn't always such a clear thing."
+    if store.mas_anni.anniCount() < 1:
```

Comment:
```suggestion
    if not store.mas_anni.pastSixMonths():
```
I'd like to have this branch for less than 6 months rather than less than one year. It seems a more natural time period for the truly exciting phase.